src/metatensor/models/utils/io.py

```diff
@@ -24,7 +23,10 @@
     pass
 
 try:
+    import rascaline
     import rascaline.torch  # noqa: F401
+
+    rascaline.set_logging_callback(lambda x, y: None)
```

Comment:
So this is problematic because it will also suppress the warnings. Maybe something like

```py
from rascaline._c_api import RASCAL_LOG_LEVEL_WARN, RASCAL_LOG_LEVEL_ERROR

def rascaline_logging(level, message):
    if level in [RASCAL_LOG_LEVEL_WARN, RASCAL_LOG_LEVEL_ERROR]:
        rascaline.log.default_logging_callback(level, message)

rascaline.set_logging_callback(rascaline_logging)
```


Another solution would be to move the spline message to `debug` instead of `info` level in rascaline.

Reply:
Ok, fair, I will do this.

But how will we do this when rascaline won't be imported explicitly in Python?

Reply:
You mean for eval? 

> Another solution would be to move the spline message to debug instead of info level in rascaline.

Reply:
Yes, I mean for eval